manifests/integreatly-cloud-resources/cloud-resources-0.5.0/cloud-resources.v0.5.0.clusterserviceversion.yaml

```diff
@@ -8,67 +8,79 @@ metadata:
           "apiVersion": "integreatly.org/v1alpha1",
           "kind": "Redis",
           "metadata": {
+            "labels": {
+              "productName": "productName"
```

Comment:
Quick question, will there be another PR for labelling our resources with the product name?